mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -109,11 +124,27 @@
 	& .wp-block-navigation-item {
 		padding-bottom: 0;
 	}
+
+	& .wp-block-navigation__submenu-icon {
+		& svg {
+			display: none;
+		}
+		background-image: url("data:image/svg+xml,%3Csvg width='10' height='7' viewBox='0 0 10 7' fill='none' stroke='white' stroke-width='1.2' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M0.416667 1.33325L5 5.49992L9.58331 1.33325'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
What are the benefits of putting this inline? I don't like it at first glance because it's not obvious what the contents are, and it can't be reused, but I'm sure there are some benefits too.